doc/BUILD

```diff
@@ -27,6 +27,13 @@ py_test(
     tags = ["exclusive", "team:ml"]
 )
 
+py_test(
```

Comment:
Should we automatically test all Python modules in `source/data/doc_code`? Looks like we do so for the other libraries:

```
py_test_run_all_subdirectory(
    size = "large",
    include = ["source/ray-air/doc_code/*.py"],
    exclude = [
        "source/ray-air/doc_code/hf_trainer.py",  # Too large
        "source/ray-air/doc_code/predictors.py",
    ],
    extra_srcs = [],
    tags = ["exclusive", "team:ml"],
)
```

Reply:
we should, but out of scope for this PR. This is being tracked in https://github.com/ray-project/ray/issues/33028

Reply:
Sweet. Didn't see that